meson.build

```diff
@@ -0,0 +1,104 @@
+project('fribidi', 'c', version: '1.0.1',
+  meson_version : '>= 0.44')
+
+# New release:
+#   interface_age++
+#   micro version++
+#
+# If any functions have been added:
+#   interface_age = 0
+#   interface_version++
+#
+# If binary backwards compatibility has been broken:
+#   panic!
+
+interface_age = 0
+interface_version = 4
```

Comment:
I don’t like the fact we will now have two places where we need to increment the version number and potentially one will be missed.

Reply:
I'm not sure what to do about the versions. I don't think there is an easy fix for this. Even if we read these two from the configure.ac (ew), the fribidi version in `project()` in the first line needs to be hard-coded there first thing.

We could derive a suitable `interface_age` and `interface_version` from your fribidi minor version and micro version, but that's going to be painful in autotools as it doesn't do simple maths. What we do in GStreamer is to use maths in the meson file and have the formula as a comment in `configure.ac`.

Another option would be to hook a script into 'make check' or 'make distcheck' that makes sure versions are consistent in `meson.build` and `configure.ac`.

I think most maintainers tend to remember that they support two build systems and fix it up in both, it's rarely been a problem :)

Reply:
> I think most maintainers tend to remember that they support two build systems and fix it up in both, it's rarely been a problem :)

You would bet :slightly_smiling_face: Anyway, I’ll differ the opinion here to @dov and @behdad as they are the actual maintainers.

Reply:
Hahaha. I never update two build systems... Anyway, @dov's call. I'm not maintainer anymore.

Reply:
How about we simply sprinkle a few `dnl keep in sync with meson.build` in `configure.ac` then? :)